__tests__/endpoints.test.js

```diff
@@ -0,0 +1,50 @@
+const app = require('../app')
+const request = require('supertest')
+const db = require('../db/connection')
+const {articleData, commentData, topicData, userData} = require('../db/data/test-data/index')
+const seed = require('../db/seeds/seed')
+
+beforeEach(() => {
+    return seed({articleData, commentData, topicData, userData})
+});
+
+afterAll(() => {
+    return db.end
+})
+
+describe('GET /api', () => {
+    it('should return a JSON object', () => {
+        return request(app).get('/api')
+        .expect(200)
+        .then((response) => {
+            console.log(response.text)
+            expect(JSON.parse(response.text)).toBeInstanceOf(Object)
+        })
+        
+    });
+    it('should contain GET /api key, api/topics and api/articles keys', () => {
```

Comment:
Not bad tests but what we can do is simply require the json file at the top of our test file and test our endpoints are returning that on the body.